Jester Edits (#488)
Giratena/Succubus-Duel-English-Translation: CSV/Language/English/Card/10057.txt

EU57_paizuriAnime,
EU57_bustShakeAnime,
EU57_footShakeAnime,

> 　道化の姿をしたモンスター。つかみどころのない態度であちこち動き回り、気が付けばこちらのそばにいてふところに潜り込んでいる。,A monster who has taken the form of a court jester.  Her elusive demeanor and unpredictable movements make her a hard target to keep track of; allowing her to sneak onto her opponent's side of the field; taking her place right beside them before they can realize what has happened.

Reviewer: ```suggestion
　道化の姿をしたモンスター。つかみどころのない態度であちこち動き回り、気が付けばこちらのそばにいてふところに潜り込んでいる。,A monster who has taken the form of a court jester.  Her elusive demeanor and unpredictable movements make her a hard target to keep track of; making it easy for her to infiltrate the opponent's side of the field; taking her place right beside them before they can realize what has happened.
```

> \nどこか憎めない愛嬌のある様子に油断すれば、その時点で彼女の手の平の上であり、主人であるはずのものがかえって召使いに操られ、いつの間にか言われるがまま精も何もかも彼女に搾り取られてしまっているだろう。,Should someone let their guard down around her seemingly lovable and charming demeanor; they'll find themselves already in the palm of her hand. Her role as servant will quickly turn into mistress; and she'll have milked them of everything they have all as you obey her every word.

Reviewer: ```suggestion
\nどこか憎めない愛嬌のある様子に油断すれば、その時点で彼女の手の平の上であり、主人であるはずのものがかえって召使いに操られ、いつの間にか言われるがまま精も何もかも彼女に搾り取られてしまっているだろう。,Should someone let their guard down around her seemingly lovable and charming demeanor; they'll find themselves in the palm of her hand before they know it. Her role as servant will quickly turn into mistress; and she'll have milked them of everything they have all as you obey her every word.
```

Author: The problem is the previous line uses an idea very similar to 'before they now it,' so I was trying to avoid doubling up.

Reviewer 2: ```suggestion
\nどこか憎めない愛嬌のある様子に油断すれば、その時点で彼女の手の平の上であり、主人であるはずのものがかえって召使いに操られ、いつの間にか言われるがまま精も何もかも彼女に搾り取られてしまっているだろう。,Should someone let their guard down around her seemingly lovable and charming demeanor; they'll quickly find themselves in the palm of her hand. Her role as servant will quickly turn into mistress; and she'll have milked them of everything they have all as you obey her every word.
```

> \n　道化として数々の「芸」を身に着けている彼女は、手コキも足コキもパイズリも得意である。ご主人の好みに合わせて、胸での奉仕を望まれれば弾力たっぷりの胸で愛撫し、足での奉仕を望まれれば絶妙な力加減でぐりぐりと踏みしごいてくれる。のちのことを考えないのであれば、頭を空っぽにし彼女に楽しませてもらうことはとても幸福なことかもしれない。,As an entertainer; she has mastered numerous different "performances." She excels at handjobs; footjobs; and paizuri; and will use whatever talent best caters to her master's preferences. If he desires service from her breasts; she'll caress him with her large; bouncy chest. Should he prefer service from her feet; she'll stroke and squeeze his cock between her arches with exquisite pressure control. If a man allows these 'performances' to make him forget about the situation at hand; he will no doubt find himsself trapped in pure bliss; head empty of all thoughts.

Reviewer: It's "performances" in the first sentence but 'performances' in the last. I personally suggest consistency with the quotation marks.

> \n　道化として数々の「芸」を身に着けている彼女は、手コキも足コキもパイズリも得意である。ご主人の好みに合わせて、胸での奉仕を望まれれば弾力たっぷりの胸で愛撫し、足での奉仕を望まれれば絶妙な力加減でぐりぐりと踏みしごいてくれる。のちのことを考えないのであれば、頭を空っぽにし彼女に楽しませてもらうことはとても幸福なことかもしれない。,As an entertainer; she has mastered numerous different "performances." She excels at handjobs; footjobs; and paizuri; and will use whatever talent best caters to her master's preferences. If he desires service from her breasts; she'll caress him with her large; bouncy chest. Should he prefer service from her feet; she'll stroke and squeeze his cock between her arches with exquisite pressure control. If a man allows these 'performances' to make him forget about the situation at hand; he will no doubt find himsself trapped in pure bliss; head empty of all thoughts.

Reviewer: ```suggestion
\n　道化として数々の「芸」を身に着けている彼女は、手コキも足コキもパイズリも得意である。ご主人の好みに合わせて、胸での奉仕を望まれれば弾力たっぷりの胸で愛撫し、足での奉仕を望まれれば絶妙な力加減でぐりぐりと踏みしごいてくれる。のちのことを考えないのであれば、頭を空っぽにし彼女に楽しませてもらうことはとても幸福なことかもしれない。,As an entertainer; she has mastered numerous different "performances." She excels at handjobs; footjobs; and paizuri; and will cater her talents to best suit her master's preferences. If he desires service from her breasts; she'll caress him with her large; bouncy chest. Should he prefer service from her feet; she'll stroke and squeeze his cock between her arches with exquisite pressure control. If a man allows these 'performances' to make him forget about the situation at hand; he will no doubt find himsself trapped in pure bliss; head empty of all thoughts.
```

> [1].このカードは「隠密」「弱点付与」を持つ。,[1]. This card has [Stealth] and [Weakness Assignment].

Reviewer: suggestion: Weakness Assignment -> Weakness Implant

Reviewer 2: Hmmm I think it's fine as is ^^

> [2].このカードが通常召喚に成功したとき、LPを800払xcv って発動する。相手フィールド上の最もレベルが低いモンスター（このカードと同名のカードは除く）とこのカードのコントロールを入れ替える。その後、そのモンスターに「攻撃不能（1ターン）」を付与する。,[2]. When this card is successfully Normal Summoned; pay 800 LP to switch control of this it with the lowest level monster on your opponent's field (excluding cards with the same name as this card). Then; inflict [Cannot Attack (1 Turn)] on that monster.

Reviewer: "this it"

わたくしめは仲間でありますから、\nたくさん甘えて大丈夫なんですよー❤ほらほらぁ、\nおっぱい吸ってもいいですしぃ、匂いをクンクンしてもいいですよぉ❤,I'm on your side here; so feel free to indulge yourself as much as you want~❤ Come now; go ahead; suck on my breasts as much as you like! You can even see how nice they smell if you want~❤
観客さまぁ、わたくしめのご奉仕をどうぞお楽しみください～❤\n道化は体ぜーんぶを使って観客さまを気持ちよくしますよぉ❤,Please enjoy your jester's service; my dear audience member~❤ I'll use my whole body to make you feel incredible~❤
ほら、服をめくって、おっぱいぴらり❤\nここでむにゅんと挟んでさしあげますね～❤\nえい、むにゅむにゅ～❤,Look; I'll lift my clothes and show off my breasts~❤ Now; let me wrap them snugly around your shaft~❤ There we go; time for a squishy massage~❤
道化めのおっぱい、なかなか形もいいと思うんですよ～❤\n玉乗りしたり、玉投げしているときも、\nたぷたぷ揺れるこのおっぱいにいつも視線を感じております～❤,My breasts are so well-shaped; don't you think?❤ Whether I'm bouncing on a ball balancing or juggling them; I can always tell when people are staring at these bouncy tits of mine~❤
観客さまもぉ、おっぱいじーっと見ておられますね❤\nほら、挟んだまま揺らしますので～、\n揺れる様子とむにむにの感触、両方お楽しみください～❤ゆさゆさぁ❤,You're ogling my breasts too; aren't you? Watch closely now~❤ I'll keep them wrapped around you and start shaking them; so make sure you enjoy both the sight of them bouncing and their super soft feel~❤ Jiggle jiggle~❤

> 主様ぁ、ご奉仕を命じてくださってありがとうございますぅ❤\n道化は体ぜーんぶを使って主様を気持ちよくしますよぉ❤,Thank so much for ordering me to serve you; Master~❤ Your jester will happily use her whole body to make you feel amazing~❤

Reviewer: ```suggestion
主様ぁ、ご奉仕を命じてくださってありがとうございますぅ❤\n道化は体ぜーんぶを使って主様を気持ちよくしますよぉ❤,Thank so much for giving me the order to serve you; Master~❤ Your jester will happily use her whole body to make you feel amazing~❤
```

あはぁ、ほんとですか～❤うれしいです、主様ぁ❤\n主様はわたくしのことを優先してくださったんですね～❤,Ahh; really?❤ I'm so happy; Master~❤ You really do care about me~❤
あるじにこんなに大事にされては、\n道化として恐悦至極です～❤\nほらぁ、うれしくてうれしくてしょうがないので～・・・❤,To have Master treat me so preciously... I'm so; so honoured!❤ Look; I'm just so happy I can't contain myself...❤
主様のお顔をおっぱいハグです❤\nだーい好きですよ、主様ぁ❤,I could just give your cute face a big boobie hug!❤ I love you so much; Master~❤
むぐっ、はううぅっ❤❤,Mmph; haaah...❤❤
あっ、あっ、だめ・・・❤❤\nもうっ、あああっ❤❤,Ah; ah; I-I can't...❤❤ Noo; aaaah!❤❤
わわわ、主様、おっぱいハグでイっちゃったんですか？,Wha- Master; did you just cum from my boobie hug?
くすくす❤敵の回し者の道化にそんなに気を許しちゃダメですよ～❤\nおっぱいの中でとろとろになって、体あずけて甘えちゃって。\n可愛らしいですね、主様ぁ❤,Kehehe~❤ You shouldn't let your guard down so easily around an enemy jester like me~❤ Getting all melty in my breasts; leaning into them and acting so spoiled. You're so adorable; Master~❤
くすくす❤道化めのおっぱいに顔を挟まれて、\nお目目がとろーんとしてしまっていますね❤,Kehehe~❤ Your face is buried between my funbags; and your eyes are glazing over~❤
もっともっとしてほしいのであれば、\nどうぞ道化めにご奉仕をお命じください❤\n道化は主様のこといくらでも気持ちよくしてあげますので～❤,If you want me to do more; all you have to do is order your jester to service you~❤ I'll make you feel as good as you want; Master~❤

> おやおやぁ❤\n観客さまはおっぱいが大好きなご様子❤\nそれなら、もっともっと道化のご奉仕をお楽しみください～❤,Oh my~❤ Big boobs sure are popular with the audience today!❤ In that case; please enjoy even more of my entertainment techniques~❤

Reviewer: ```suggestion
おやおやぁ❤\n観客さまはおっぱいが大好きなご様子❤\nそれなら、もっともっと道化のご奉仕をお楽しみください～❤,Oh my~❤ Big boobs sure are popular with the audience today!❤ In that case; please enjoy even more of my entertaining techniques~❤
```

くすくす❤そう言われても、わたしに密着されて\n横でささやかれていたら、オナニー止められませんか？\nどんどん手が早くなっておりますね❤観客さま❤,Kehehe~❤ Even after I say that; when I'm pressed this close to you; whispering in your ear; you can't stop masturbating; can you? Your hand is moving faster and faster~❤ My poor; dear audience member~❤
それではちょっと玉遊びでもいたしましょうか。\n片手を観客さまのタマタマにあてがいまして、\nほら、ふにふに♪もみもみもみ♪,Now then; how 'bout I play with your balls for a bit? I'll place one hand on your testicles and... Squish; squish~♪ Rub rub rub~♪
道化めの手先は器用でしょう？わたくし玉の扱いはお手の物ですので～\n観客さまの玉も上手に扱えるのです❤\nオナニーの手伝いでもみもみされて気持ちいいですね～❤,A jester's hands are quite dexterous; you know. I'm an expert at handling balls; so I can play with yours no problem; too.❤ Having them massaged as I aid our masturbation feels amazing; doesn't it?❤
うあっ、ああうっ❤すごいっ、きもちいいっ❤\nあっ、そんなにぐにぐにされたらっ、はひぃっ❤,Uah; aah!❤ Incredible. It feels so good!❤ Ah; if you keep kneading them like that... Hahii!❤
くすくす❤観客さまは道化の芸に大満足のご様子❤\nもっともっとその歓声をお聞かせくださいまし❤,Kehehe~❤ It seems the audience member is thoroughly satisfied with this jester's performance~❤ Please; let me hear more of those delightful sounds~❤

> 我慢ができなくなったらいつでも出してよいのですよ♪\nほら、おちんちんの前にもう片方の手をかぶせてあげますので、\nお好きな時に道化めの手の平にお出しください❤,Whenever you can't hold back anymore; feel free to let it all out!♪ Look; I'll cup my other hand in front of your cock; so when you're ready; release it all and I'll catch it like this.❤

Reviewer: ```suggestion
我慢ができなくなったらいつでも出してよいのですよ♪\nほら、おちんちんの前にもう片方の手をかぶせてあげますので、\nお好きな時に道化めの手の平にお出しください❤,Whenever you can't hold back anymore; feel free to let it all out!♪ Look; I'll cup my other hand in front of your cock; so when you're ready; just release it all; and I'll catch it like this.❤
```

耳元でささやかれながら玉ももみもみ揉まれて❤\nとっても贅沢なオナニーですね～❤\nくすくす❤,Having your ear teased with whispers while your balls are kneaded and massaged...❤ What a luxurious masturbation session; huh?❤ Kehehe~❤
それではどうぞ、おひねり代わりに、\n道化めに観客さまのエナジーをお恵みくださいませ～❤\nふぅー・・・❤,Now then; go ahead and bless this jester with your energy as a tip; dear audience member~❤ Haah...❤
きゃは❤どくどくどくっと、濃ゆいのが出ております❤\n観客様の子種が道化めの手の平の上に、こんなにたくさん❤,Kyaha!❤ It's spurting out in thick; heavy pulses!❤ So much of my audience member's seed; all over this jester's palm~❤
道化めのサポートはいかがだったでしょうか？\n存分に楽しませることができたのなら、道化冥利に尽きるというもの♪\nくす、満喫しちゃった顔ですね、観客さま❤,I did a good job aiding you; didn't I? If I managed to truly entertain you; then as a jester; I couldn't be happier!♪ Kehehe; you're making such a satisfied expression; dear audience member~❤

> おやおや、主様もオナニーをされてしまうのですか？\nくす、命じてくださればご奉仕をいくらでもいたしますのに～❤\n道化めの体はそんなにエッチだったでしょうか❤,Oh my; is Master going to masturbate; too? Kehehe; if you had just given the order; I would have gladly serviced you as much as you want~❤ Is your jester's body really that sexy?❤

Reviewer: ```suggestion
おやおや、主様もオナニーをされてしまうのですか？\nくす、命じてくださればご奉仕をいくらでもいたしますのに～❤\n道化めの体はそんなにエッチだったでしょうか❤,Oh my; is Master going to masturbate; too? Kehehe; if you had just given the order; I would have gladly serviced you as much as you want~❤ Is your jester's body really so desirable?❤
```

> では、主様もお仲間さまと同じ様に、\nオナニーのお手伝いをしてあげますね～❤\n主様のお耳をお借りしまして、ふぅー・・・❤,Well then; I'll help Master with his masturbation; just like I did for your friend~❤ Allow me to borrow Master's ear for a moment. Haah...❤

Reviewer: > Well then; I'll help Master with his masturbation; just like I did for your friend~❤

Make "his maturbation" and "his friend", or "your masturbation" and "your friend"

道化の芸では、息なんて普段は使わないのですよ？\nせいぜい紙吹雪をふぅーっと飛ばすときぐらいです。\n主様の理性も、紙吹雪のようにふぅっと吹き飛ばされてしまいますか？,In my performances; I don't normally use my breath; you know. The most I do is blow confetti with a puff. I wonder... Will Master's reason be blown away as easily as confetti?
道化に仲良く体を寄せてもらいながら、しこしこ❤しこしこ❤\nお仲間のマスターたちや多くの人間の命がかかっていますのに、\n耳に息をふぅふぅ吹きかけられて、頭が真っ白❤気持ちいい❤,Stroke; stroke; stroke~❤ That's it; keep stroking yourself as your jester presses tight against you~❤ Even though the lives of your fellow cardmasters and so many other humans are at stake; having breath blown into your ear just makes your mind go blank~❤ It feels so good~❤
おや、向こうで主様の仲間のマスターたちが何やら言っておりますね～。\nきっと主様を正気に戻そうと声を張り上げているのでしょう❤\nではでは、余計な言葉がお耳に入らぬように、ほら、ふぅーー・・・❤,Oh my; it looks like your fellow cardmasters over there are trying to tell you something. They're probably shouting; trying to bring you back to your senses.❤ Well then; to make sure those unnecessary words don't reach your ears; here. Haah...❤

> 道化めの声と息だけで脳みそを埋めてあげます❤\n何も考えず、おちんちんをお扱きください、主様ぁ❤\nそうそう上手ですよ～、シコシコ❤シコシコ❤さすが主様です❤,I'll fill your mind with nothing but your sweet jester's voice and breath.❤ Don't think about anything else; Master. Just keep stroking your cock~❤ That's it. You're doing so good. Stroke; stroke~❤ Stroke; stroke~❤ I'd expect nothing less of my master!❤

Reviewer: ```suggestion
道化めの声と息だけで脳みそを埋めてあげます❤\n何も考えず、おちんちんをお扱きください、主様ぁ❤\nそうそう上手ですよ～、シコシコ❤シコシコ❤さすが主様です❤,I'll fill your mind with nothing but your sweet jester's voice and breath.❤ Don't think about anything else; Master. Just keep stroking your cock~❤ That's it. You're doing so well. Stroke; stroke~❤ Stroke; stroke~❤ I'd expect nothing less of my master!❤
```

おやおや、お仲間さまがされているのはオナニーですか？\nくす、みなさま揃って道化めでオナニーをして、\nわたしの体はそんなにエッチだったでしょうか～❤,Oh gosh; are you maturbating; too; dear comrade? Kehehe; everyone's jerking off to this jester together! Was my body really that sexy?❤
お仲間さまぁ❤あなたのオナニーも道化めがお手伝いいたします❤\nオナニーがとっても気持ちよくなるように、\nお仲間さまのお耳をお借りしまして、ほら、ふぅー・・・❤,My poor; sweet ally~❤ I know; I'll help you masturbate; too!❤ To make it feel extra good; allow me to borrow your ear for a moment. Here; haah...❤
くすくす❤敏感な反応ですねぇ、お仲間さまぁ❤\nみなさんお耳が弱点なのですか～❤\n軽く息を吹きかけただけでお顔がとろけてしまいます❤,Kehehe~❤ You're so sensitive; my friend~❤ Are all of you weak to having your ears played with?❤ Just a light puff of my breath and your expression just melts completely~❤
玉投げに綱渡り、ナイフ投げに空中ブランコ、\n色んな芸でお客様を楽しませてきましたが、\n息を吹きかけるだけなんて、こんなに簡単な芸は他にありませんね❤,Ball juggling; tightrope walking; knife throwing; trapeze acts—I've entertained audiences with all kinds of performances; but blowing my breath? There's no easier trick in the book~❤

> 何の技術もいりません。ただシコシコしている横で、\n時折、ふぅーっと息を吐いてあげるだけでいいのです❤\nそれだけで、くす、簡単に至福の表情♪,It requires no skill whatsoever. All I havd to do is blow a gentle breath while you stroke yourself next to me~❤ And just like that; kehehe; you make such a blissful expression~♪

Reviewer: ```suggestion
何の技術もいりません。ただシコシコしている横で、\n時折、ふぅーっと息を吐いてあげるだけでいいのです❤\nそれだけで、くす、簡単に至福の表情♪,It requires no skill whatsoever. All I have to do is blow gently while you stroke yourself next to me~❤ And just like that; kehehe; you make such a blissful expression~♪
```

> 道化めの手にかけることができて、いかがでしたか主様❤\nお耳が弱いだなんてかわいいところもあるのですね❤\nくすくす❤,How was it; cumming all over your jester's hands; Master?❤ To think you would have such a cute weakness for having your ears teased~❤ Kehehe~❤

Reviewer: ```suggestion
道化めの手にかけることができて、いかがでしたか主様❤\nお耳が弱いだなんてかわいいところもあるのですね❤\nくすくす❤,How was it; cumming all over your jester's hands; Master?❤ To think you would have such a cute weakness towards having your ears teased~❤ Kehehe~❤
```

Author: don't like that. Feels weird.

あん、足裏で勢いよく射精しています❤\n足ですりすりと擦られて我慢できませんでしたか？\n射精させるのは玉乗りするより簡単ですね～❤くすくす❤,Mmm; you're ejaculating so vigorously against the soles of my foot~❤ Couldn't hold back while being rubbed by my arch; could you? Making you cum is even easier than riding a ball! Kehehe~❤
主様ぁ、どうされましたか？もう体が動かないのですか？\nおよよよ、道化はやりすぎてしまったようです❤,Awh; what's wrong; Master? Can't move your body anymore? Oh my; it looks like your jester went too far~❤
これぐらいなら耐えられると思いましたのに、\n気持ちよくしすぎてしまって、ごめんなさいです、主様ぁ❤\nぷっ、くすくすくす❤,I thought you could endure at least this much; but I made you feel too good. Sorry about that; Master~❤ Pfft; kehehehehe~❤
ふふ、何やってるのかしら、{主人公}。\n道化師は敵のモンスターを奪う強力なモンスターだけれど、そっちに\nコントロールが移ってからはちゃんとあなたの命令を聞くはずよ？,Fufu; what are you doing; {主人公}? The jester might be a powerful card that steals enemy monsters; but once control shifts to your side; she should obey your commands like any other monster; right?
自分から道化師にエッチなことを命じて、勝手に負けて、\nカードマスター失格ね❤,You gave the jester those lewd commands yourself and lost on your own accord. That should be disqualifing as a cardmaster; don't you think?❤

> あなたが負けたらみんな吸い殺されちゃうかもしれないのに、\nふふふ、わたしから溢れているサキュバスの淫気に当てられて、\n理性を失っちゃったのかしら❤,If you lose; everyone might get sucked dry; you know. And yet; fufufu; maybe my succubus aura's just driven you crazy~❤

Reviewer: ```suggestion
あなたが負けたらみんな吸い殺されちゃうかもしれないのに、\nふふふ、わたしから溢れているサキュバスの淫気に当てられて、\n理性を失っちゃったのかしら❤,If you lose; everyone might get sucked dry; you know. And yet; fufufu; maybe my succubus aura's simply driven you mad~❤
```

うう、そうだ、僕は何をやっているのだろう。いまは欲望に流されていい状況じゃない。サキュバスを前にして命がかかっているのだ。,Ugh; that's right; what am I doing? This isn't a situation where I can give in to desire. I'm facing a succubus. Lives are at stake.

> だが、エルルさんがサキュバス化してから、狭い地下通路の中に甘ったるい空気が漂っている。その匂いを嗅いでいると体がむらむらして仕方がなくなる。,But ever since Elulu turned into a succubus; a sickeningly sweet atmosphere had been drifting through the narrow underground passage. Breathing it in for so long had excited me so much I couldn't think straight.

Reviewer: ```suggestion
だが、エルルさんがサキュバス化してから、狭い地下通路の中に甘ったるい空気が漂っている。その匂いを嗅いでいると体がむらむらして仕方がなくなる。,But ever since Elulu turned into a succubus; a sickeningly sweet atmosphere has been drifting through the narrow underground passage. Breathing it in for so long got me so excited that I couldn't think straight.
```

Author: Narration is always past tense.

Reviewer 2: ```suggestion
だが、エルルさんがサキュバス化してから、狭い地下通路の中に甘ったるい空気が漂っている。その匂いを嗅いでいると体がむらむらして仕方がなくなる。,But ever since Elulu turned into a succubus; a sickeningly sweet atmosphere had been drifting through the narrow underground passage. Breathing it in for so long got was making me so aroused that I couldn't think straight.
```

Author: Giratena, the change you made has artifacts. I think 'got' is a remnant from an alternative wording that needs to be deleted

Reviewer 2: Good catch, thank you...

> そのうえ、自分のそばにやってきた、この怪しい女の子が、目の前でぴょんぴょんと跳ねて誘惑してくる。密着しそうな距離で、わざとらしくお尻や胸を揺らして誘惑してきて、それで・・・。,On top of that; this suspicious girl who'd come close to me was bouncing up and down in front of my face; tempting me. She'd deliberately swayed her ass and breasts so close to me; trying to seduce me. Because of that...

Reviewer: ```suggestion
そのうえ、自分のそばにやってきた、この怪しい女の子が、目の前でぴょんぴょんと跳ねて誘惑してくる。密着しそうな距離で、わざとらしくお尻や胸を揺らして誘惑してきて、それで・・・。,On top of that; this suspicious girl who'd approached me was bouncing up and down in front of my face; tempting me. She'd deliberately swayed her ass and breasts so close to me; trying to seduce me. Because of that...
```

しかし、決して僕の体に触れることはない。ほんのわずかの距離を開けて、彼女の体がかすめていく。にやにやと笑いながら、僕を試すように、いやらしく体を動かしてくる。,However; she never actually touched my body; always maintaining just the slightest distance; her body brushing past mine. Grinning mischievously; as if testing me. She continued to move her body in lewd; enticing steps.
うあっ、ううっ❤だめっ、これ以上はっ❤\nああっ・・・❤,Uah; ungh.❤ No; I-I can't take it any more!❤ Aah...❤
今からでも理性を取り戻し、エルルさんと戦わないといけない。僕以外にも何人もカードマスターがいる、みんなで力を合わせればもしかしたら、魔力を多く失った今の状態でも勝てるかもしれない。,I need to come to my senses again and fight Elulu. There are so many other cardmasters here with me—if we all work together; maybe we can still win; despite the fact I've lost so much magical power.
だから、この目の前の女の子にやめるよう命じて、カードを引かないといけないのだ。そう、理性を振り絞って気持ちいいことなど忘れなくては・・・。,That's why... That's why I need to order this girl in front of me to stop so I can draw my cards. That it; I just need to muster my will and forget about feeling good...
主様ぁ、悩んでいるなら、\nどうぞ❤これを御覧になってくださいまし❤,Awh; Master. You seem conflicted~❤ Then maybe I can help you. Please; watch this~❤

> 主様のおちんちんの前で、道化の得意技、玉投げでございます❤\n胸を両手で持ちまして～、ほらぁ、\nたぷたぷたぷと左右交互に揺らして見せるのです❤,I'll get right in front of Master's cock and show you your jester's specialty—ball juggling~❤ I'll hold my breasts with both hands; and... Look; I'll bounce them for you; alternating left and right~❤

Reviewer: ```suggestion
主様のおちんちんの前で、道化の得意技、玉投げでございます❤\n胸を両手で持ちまして～、ほらぁ、\nたぷたぷたぷと左右交互に揺らして見せるのです❤,I'll step right in front of Master's cock and show you this jester's specialty—ball juggling~❤ I'll hold my breasts with both hands; and... Look; I'll bounce them for you; alternating left and right~❤
```

Author: 'Step' isn't right because she's not stepping there. If you wanted to make a change from 'get,' you could do 'kneel,' but I don't feel like either of these changes are necessary.

Reviewer 2: I like kneel, lets use that ^^

```suggestion
主様のおちんちんの前で、道化の得意技、玉投げでございます❤\n胸を両手で持ちまして～、ほらぁ、\nたぷたぷたぷと左右交互に揺らして見せるのです❤,I'll kneel right in front of Master's cock and show you your jester's specialty—ball juggling~❤ I'll hold my breasts with both hands; and... Look; I'll bounce them for you; alternating left and right~❤
```

さっきからあなたが一方的に負けてるのを見て、\nみんなおびえてるの。あの無敗のルーキーがぼろぼろにされてるんだもの、\nきっとわたしの強さが怖いのね。,Everyone's so scared after watching you lose so helpleslly. The undefeated rookie getting torn apart like this—they must be terrified of just of how strong I am.
大丈夫、すぐにみんなの恐怖を快楽に変えてあげるから❤\nそれじゃ、{主人公}❤せいぜいその子と遊んでなさい❤くすくす❤,Don't worry; I'll turn everyone's fear into pleasure soon enough~❤ Well then; {主人公}~❤ Have fun playing with that girl~❤ Kehehe~❤
ああっ、だめなのにっ❤こんなことしてる場合じゃないのにっ❤\nううっ❤きもちよくて、あああっ❤,Aah; I-I shouldn't be doing this!❤ It isn't the time for this!❤ Ungh...❤ But it feels so good; aaah...❤
にひひ❤主様は難しいことなど考えなくてよいのですよ❤\n道化の仕事は、難しいことを忘れさせて、一時のバカ騒ぎに\n主様を陶酔させることです❤,Ehehe~❤ You needn't think about such difficult things; Master~❤ My job as a jester is to make you forget all about those complicated little matters by keeping you happy~❤
おっぱいでずりずりされて、\n主様のここはすっかり難しいことを考えられなくなっています❤\n頭の方ももっと空っぽにしちゃいましょうよ❤主様ぁ❤,Your cock can't think about difficult things while it's being rubbed between my breasts down here~❤ Let's empty out all those thoughts; okay Master?~❤
ほらぁ、こうやってどんどん早くしていきますよぉ❤\n道化めの技をご堪能くださいまし❤,Look; I'll go faster and faster; like this~❤ Please; savor your jester's technique~❤

> くすくす❤動きを早くしたら、腰がビクビクと跳ねだしてしまいました❤\n必死に快楽を逃がそうとしてるのか、\nそれとも快楽を貪っているのか、道化にはわかりませぬ❤,Kehehe~❤ When I sped up my movements; your hips started twitching like crazy.❤ Are you desperately trying to escape the pleasure; or are you greedily looking for more? This jester just can't tell~❤

Reviewer: ```suggestion
くすくす❤動きを早くしたら、腰がビクビクと跳ねだしてしまいました❤\n必死に快楽を逃がそうとしてるのか、\nそれとも快楽を貪っているのか、道化にはわかりませぬ❤,Kehehe~❤ When I sped up my movements; your hips started twitching like crazy.❤ Are you desperately trying to escape the pleasure; or are you greedily demanding more? This jester just can't tell~❤
```

Author: I feel like this change detracts from the line. 'Looking for more' implies a level of desperation without control, where 'demanding' has that implication of control, you know?

Reviewer 2: I agree. Let's go with this:
```suggestion
くすくす❤動きを早くしたら、腰がビクビクと跳ねだしてしまいました❤\n必死に快楽を逃がそうとしてるのか、\nそれとも快楽を貪っているのか、道化にはわかりませぬ❤,Kehehe~❤ When I sped up my movements; your hips started twitching like crazy.❤ Are you desperately trying to escape the pleasure; or are you genuinely seeking more? This jester just can't tell~❤
```

なんて、もう意識もないでしょうけども❤\n主様が道化とバカ騒ぎしている間に、\n向こうのマスターたちもみな片付いたようです❤,Awh; but it seems you're already losing consciousness~❤ It looks like Elulu's dealt with all the other cardmasters while we were fooling around; too.❤
エルル様にカードに戻してもらったら、\nそれでわたしと主様の関係はおしまいですね。\n一時の主従関係楽しゅうございましたよ、主様❤,Once Lady Elulu returns me to card form; I won't be your servant anymore. But I really enjoyed serving under you; Master~❤
僕は薄れていく意識の中で、道化師がエルルさんのカードに戻っていくのを見た。僕はそれを見て、悔しがるでも恨めしがるでもなく、ただ「もっと、おねがい・・・」と無意識につぶやいていた。,Through my fading consciousness; I watched the jester return to Elulu's card. But I didn't feel frustration; nor resentment. All I wanted was to beg her to pleasure me again...
死ぬ手前まで干からびている僕を、エルルは一瞥し、そして軽蔑したようなまなざしで見ながら、僕に一度だけ口づけをした。そのまま最後に残っていたエナジーを吸いつくされ・・・,Elulu glanced at me as I lay there; drained and barely alive; gazing at me with what seemed like contempt; before giving me a kiss and draining the last of my energy from me...
僕は暗い地下通路の中力尽きたのだった。僕の物語は、こうしてサキュバスに吸い殺されて、その幕を閉じた。,With that; I collapsed in the dark underground passage. And so; my story ends here; drained dry by a succubus.

> あは、道化のようなザコモンスターが、\n観客さまに勝ってしまいました❤,Aha; a little monster as weak as a jester actually defeated an audience member!❤

Reviewer: ```suggestion
あは、道化のようなザコモンスターが、\n観客さまに勝ってしまいました❤,Aha; a little monster as weak as this jester actually defeated an audience member!❤
```

Author: Feel like this change is unnecessary.

フィナーレは早く振動させながらこすりあげますね❤\n我慢などで出来ぬよう、搾りだしますゆえ❤\nどうぞ、ほら、びゅっ、びゅっ、びゅっ～❤,For the finale; I'll rub you quickly with rapid vibrations.❤ I'll milk every last drop out so you can't hold back anymore.❤ Come on now; hurry up and spurt; spurt; spurt~❤
にひひ❤たくさん出しましたね、観客さま❤\nおや、いっぱい出したせいで、体がだいぶやつれてきているご様子❤\nエナジーが失われてだいぶ苦しいのではないですか？,Ehehehe!❤ You came so much; dear audience member.❤ Oh my; after releasing all that; your body looks completely drained.❤ You must be suffering now that so much of your energy leaked out; right?
ですが、苦しいのとは別に、頭の奥がじんと痺れたように\n心地よいのですよね～❤サキュバスに吸われるときは\n得てしてそういうもの❤抵抗する気力は快楽に奪われるのです❤,But you're not suffering at all; are you? You can feel your mind gettin foggy and sleepy; can't you?❤ That's just what it's like when a succubus drains you.❤ The pleasure steals away your will to resist~❤
自分の命がかかっていても、にひひ、足でぐりぐりされるだけで、\n自分から望んで精を吐き出してしまいます❤道化のようなたいした\n戦闘力もないモンスターでも強いマスターを踏みにじれてしまいます❤,Even though your life is on the line... Pfft; just having my feet grind against your crotch makes you willingly offer up all your cum.❤ Even a monster like this jester with no real combat ability can trample a strong master like you~❤

> さささ、このまま意識が途切れるまで、\n道化めの技をお楽しみくださいまし❤\n快楽の中、死んで、干からびて、みじめに終わってくださいまし❤,Now then; please enjoy this jester's techniques until your consciousness fades away.❤ Die in pleasure; shrivel up; and meet your miserable end~❤

Reviewer: ```suggestion
さささ、このまま意識が途切れるまで、\n道化めの技をお楽しみくださいまし❤\n快楽の中、死んで、干からびて、みじめに終わってくださいまし❤,Now then; please enjoy this jester's techniques until your consciousness fades away.❤ Die in bliss as you shrivel up and meet your miserable end~❤
```

> 残酷ですか？サキュバス相手に腰を突き出して、\n無抵抗でいる観客さまが悪いのですよ❤\n食べてくれとねだっているようなものですゆえ❤,Is that kinda cruel? It's your own fault for thrusting your hips toward a succubus without resisting.❤ You're practically begging to be devoured~❤

Reviewer: ```suggestion
残酷ですか？サキュバス相手に腰を突き出して、\n無抵抗でいる観客さまが悪いのですよ❤\n食べてくれとねだっているようなものですゆえ❤,Do you think that's cruel of me? It's your own fault for thrusting your hips toward a succubus without resisting.❤ You're practically begging to be devoured~❤
```

\n　道化として数々の「芸」を身に着けている彼女は、手コキも足コキもパイズリも得意である。ご主人の好みに合わせて、胸での奉仕を望まれれば弾力たっぷりの胸で愛撫し、足での奉仕を望まれれば絶妙な力加減でぐりぐりと踏みしごいてくれる。のちのことを考えないのであれば、頭を空っぽにし彼女に楽しませてもらうことはとても幸福なことかもしれない。,As an entertainer; she has mastered numerous different "performances." She excels at handjobs; footjobs; and paizuri; and will use whatever talent best caters to her master's preferences. If he desires service from her breasts; she'll caress him with her large; bouncy chest. Should he prefer service from her feet; she'll stroke and squeeze his cock between her arches with exquisite pressure control. If a man allows these 'performances' to make him forget about the situation at hand; he will no doubt find himsself trapped in pure bliss; head empty of all thoughts.
[1].このカードは「隠密」「弱点付与」を持つ。,[1]. This card has [Stealth] and [Weakness Assignment].

> [2].このカードが通常召喚に成功したとき、LPを800払xcv って発動する。相手フィールド上の最もレベルが低いモンスター（このカードと同名のカードは除く）とこのカードのコントロールを入れ替える。その後、そのモンスターに「攻撃不能（1ターン）」を付与する。,[2]. When this card is successfully Normal Summoned; pay 800 LP to switch control of this it with the lowest level monster on your opponent's field (excluding cards with the same name as this card). Then; inflict [Cannot Attack (1 Turn)] on that monster.

Reviewer 2: ```suggestion
[2].このカードが通常召喚に成功したとき、LPを800払xcv って発動する。相手フィールド上の最もレベルが低いモンスター（このカードと同名のカードは除く）とこのカードのコントロールを入れ替える。その後、そのモンスターに「攻撃不能（1ターン）」を付与する。,[2]. When this card is successfully Normal Summoned; pay 800 LP to switch control of this card with the lowest level monster on your opponent's field (excluding cards with the same name as this card). Then; inflict [Cannot Attack (1 Turn)] on that monster.
```

Author: So here I think 'it' fits better just because of the repetition of 'this card' is kind of a lot in this line.

Reviewer 2: Hmm I agree.... I'll resolve.

> [3].このカードのコントローラーが元々の持ち主と違う場合、このカードは相手キャラクターに攻撃できず、自分キャラクターに対してのみ攻撃できる。このときこのカードは反撃されない。,[3].If the controller of this card doesn't own it; this card cannot attack opponent characters and can only attack your own characters. It does not receive counterattacks when doing so.

Reviewer 2: I feel like this change is a flow regression.

Author: Yeah that's what I was afraid of. I'm trying to find a better way of saying that first part because it feels a little strange as is, but I keep bumping into walls with it -.-

Reviewer 2: Hmm why not revert it?

Author: So when I go to fix things I will, 8 just wish there was a less wordy way to say it

Reviewer 2: Hmm how about this?
```suggestion
[3].このカードのコントローラーが元々の持ち主と違う場合、このカードは相手キャラクターに攻撃できず、自分キャラクターに対してのみ攻撃できる。このときこのカードは反撃されない。,[3]. If you are not the owner of this card; you cannot use this card to declare an attack on any foe. If this card is used to attack cards on your field; it does not receive counterattacks.
```

> [4].このカードのコントローラーが元々の持ち主と違う場合、自分ターン終了時に発動する。相手はカードを１枚ドローする。,[4].If the controller of this card doesn't own it; your opponent draws 1 card at the end of your turn. 

Reviewer 2: Same here

ずりずり、むにむにゅ❤あん、おチンポが震えてますぅ❤\nこんなに感動していただいて、道化めはとっても嬉しいですぅ❤\nわたくしとの玉遊びで、もっともっと元気になってくださぁい❤,My lovely titties are sliding so effortlessly up and down the length of your cock~❤ Ahn; I can feel it trembling!❤ I'm so happy you're enjoying it so much~❤ Go ahead; just have fun and buck your hips into my funbags~❤
ご命令くださりありがとうございます❤主様ぁ❤\n道化めがこのおっぱいでたくさん気持ちよくしちゃいますね❤\n玉投げに玉乗り、道化は丸いものの扱いはお手の物なのです❤,Thanks for the orders; Master!❤ Your jester will happily use her breasts to make you feel incredible~❤ Ball juggling; ball riding—jesters like me are experts at handling all kinds of round things!❤
むにゅっと棒を挟ませていただきまして～❤\n両手で交互にたぱん、たぱんっと持ち上げるのです❤主様のおチンポ、\nおっぱいの間で翻弄されてぇ、気持ちよさそうですね～❤,Let me wrap them around your shaft~❤ I'll lift them with both hands; bouncing them one at a time; up and down.❤ Master's cock is being toyed with between my breasts! It feels so good; doesn't it?❤
ずりずり、むにむにゅ❤あん、おチンポが震えてますぅ❤\nこんなに武者震いされて、主様ぁ❤戦いのやる気はばっちりですねぇ❤\nわたくしとの玉遊びで、もっともっと元気になってくださぁい❤,Sliding; squishing~❤ Ahn; your cock's trembling!❤ Master; you're shaking so much~❤ You must be so full of fighting spirit!❤ Go ahead and get even more pumped up from playing with my round; bouncy tits~❤

> 主様のご命令ですので、お仲間さまにおっぱいでのご奉仕です～❤\n玉投げに玉乗り、丸いものの扱いはお手の物なのです。当然、道化めの\n胸についたこの丸ーい二つの玉も、巧みに扱ってみせますよ～❤,Since you've given the order; Master; I'll happily pleasure your friend with my breasts~❤ Ball juggling; ball riding—I'm an expert at handling all kinds of round things. Naturally; that also includes these two round; bouncy tits of mine~❤

Reviewer 2: This changes the POV from the targeted monster to the MC. This might not be something we want to do?

Author: Honestly, the original version had a bit of confusing POV that made it seem like it was switching between the two XD

Reviewer 2: Hmm the POV has always been the target of the attack, so let's revert this for consistency.

くすくす❤道化めに甘えるその姿、\nかわいらしいですね～❤\nおチンポもたくさんなでてあげますね～❤しこしこ❤しゅこしゅこ❤,Kehehe~❤ You look so adorable cuddling up to this jester~❤ I'll stroke your cock plenty for you~❤ Pump pump; stroke stroke~❤
わたくしめは仲間でありますから、\nたくさん甘えて大丈夫なんですよー❤ほらほらぁ、\nおっぱい吸ってもいいですしぃ、匂いをクンクンしてもいいですよぉ❤,I'm on your side here; so feel free to indulge yourself as much as you want~❤ Come now; go ahead; suck on my breasts as much as you like! You can even see how nice they smell if you want~❤
観客さまぁ、わたくしめのご奉仕をどうぞお楽しみください～❤\n道化は体ぜーんぶを使って観客さまを気持ちよくしますよぉ❤,Please enjoy your jester's service; my dear audience member~❤ I'll use my whole body to make you feel incredible~❤
ほら、服をめくって、おっぱいぴらり❤\nここでむにゅんと挟んでさしあげますね～❤\nえい、むにゅむにゅ～❤,Look; I'll lift my clothes and show off my breasts~❤ Now; let me wrap them snugly around your shaft~❤ There we go; time for a squishy massage~❤
道化めのおっぱい、なかなか形もいいと思うんですよ～❤\n玉乗りしたり、玉投げしているときも、\nたぷたぷ揺れるこのおっぱいにいつも視線を感じております～❤,This jester's breasts are so well-shaped; don't you think?❤ Whether I'm bouncing on a ball balancing or juggling them; I can always tell when people are staring at these bouncy tits of mine~❤

> 観客さまもぉ、おっぱいじーっと見ておられますね❤\nほら、挟んだまま揺らしますので～、\n揺れる様子とむにむにの感触、両方お楽しみください～❤ゆさゆさぁ❤,You're ogling breasts too; aren't you? Watch closely now~❤ I'll keep them wrapped around you and start shaking them; so make sure you enjoy both the sight of them bouncing and the soft sensation~❤ Jiggle jiggle~❤

Reviewer 2: ```suggestion
観客さまもぉ、おっぱいじーっと見ておられますね❤\nほら、挟んだまま揺らしますので～、\n揺れる様子とむにむにの感触、両方お楽しみください～❤ゆさゆさぁ❤,You're ogling my breasts too; aren't you? Watch closely now~❤ I'll keep them wrapped around you and start shaking them; so make sure you enjoy both the sight of them bouncing and the soft sensation~❤ Jiggle jiggle~❤
```
